website_baidu_map/website_baidu_map.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+    <data>
+        <template id="company_description_inherited" name="map" inherit_id="website.company_description" customize_show="True">
+            <xpath expr="a" position="replace">
+                <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+                <meta name="viewport" content="initial-scale=1.0, user-scalable=no" />
+                <style type="text/css">
+                    body, html {width: 100%;height: 100%;margin:0;font-family:"微软雅黑";}
+                    #allmap{width:100%;height:250px;}
+                    p{margin-left:5px; font-size:14px;}
+                </style>
+                <script type="text/javascript" src="http://api.map.baidu.com/api?v=1.3"></script>
```

Comment:
Why use the version 1.3? Is this version too old since it go live on 2012.9.3? 

(http://developer.baidu.com/map/index.php?title=jspopular/theupdatelog)